dd-sdk-android/src/test/kotlin/com/datadog/android/rum/internal/domain/event/RumEventSerializerTest.kt

```diff
@@ -87,27 +77,31 @@ internal class RumEventSerializerTest {
                 hasField("id", event.view.id)
                 hasField("url", event.view.url)
             }
-            .hasField("usr") {
+            .hasField("_dd") {
+                hasField("format_version", 2L)
+            }
+        event.usr?.let {
+            assertThat(jsonObject).hasField("usr") {
                 hasNullableField("id", event.usr?.id)
                 hasNullableField("name", event.usr?.name)
                 hasNullableField("email", event.usr?.email)
+                containsAttributes(it.additionalProperties)
             }
-            .hasField("_dd") {
-                hasField("format_version", 2L)
+        }
+        event.context?.additionalProperties?.let {
```

Comment:
just for the consistency with the block above

```suggestion
        event.context?.let {
```

Reply:
Hmm not sure if I understand, here I am applying the assertions only if the `additionalProperties` field exists, otherwise it will not exist in the `json` object. Above in the `usr` object that field is not an optional so it will be an empty map by default.